index.html

```diff
@@ -1012,7 +1012,7 @@ <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="cont
      <dt>Feedback:
      <dd><span><a href="mailto:public-webappsec@w3.org?subject=%5BREFERRER%5D%20YOUR%20TOPIC%20HERE">public-webappsec@w3.org</a> with subject line “<kbd>[REFERRER] <i data-lt="">… message topic …</i></kbd>” (<a href="http://lists.w3.org/Archives/Public/public-webappsec/" rel="discussion">archives</a>)</span>
      <dt>Issue Tracking:
-     <dd><a href="https://github.com/jeisinger/webappsec-referrer-policy/issues/">GitHub</a>
+     <dd><a href="https://github.com/estark37/webappsec-referrer-policy/issues/">GitHub</a>
```

Comment:
this really should be w3c :-/

@mikewest any idea what to do about this? just ignore?